cernopendata/modules/fixtures/data/records/cms-dimuon-spectrum-2011.json

```diff
@@ -58,7 +58,7 @@
low the instructions in step 1 at <a href=\"/docs/cms-virtual-machine-2011\">How to install a CERN Virtual Machine</a>.</p> <p>You can setup the example with the detailed instructions below, if you want to get insight on how to build your own analysis and what are the files needed. Install and run the Demo (demo analyzer) program as instructed in <a href=\"/docs/cms-virtual-machine-2011#test\">step 2 at 'How to install a CERN Virtual Machine'</a>. Then <ol> <li>Create directory <code>datasets</code> under <code>Demo/DemoAnalyzer</code></li>\n <li>Download the index files for the <a href=\"/record/17\">/DoubleMu/Run2011A-12Oct2013-v1/AOD</a> primary dataset (default) to the <code>Demo/DemoAnalyzer/datasets</code> directory</li>. Only the first index file is actually needed for initial running. You can also use other primary datasets containing muons from 2011 or 2012 as listed under "Use With" above.</li>\n <li>Download <a href=\"/record/1001\">the 2011 JSON validation file</a> (default) or <a href=\"/record/1002\">the 2012 JSON validation file</a> (if needed) to the <code>Demo/DemoAnalyzer/datasets</code> directory</li>.\n <li>Replace the three files <code>BuildFile.xml</code>, <code>demoanalyzer_cfg.py</code>, <code>src/DemoAnalyzer.cc</code> with the ones from this record and read the comments in <code>DemoAnalyzer.cc</code> for more details.</li>\n <li>Recompile (<kbd>scram b</kbd>) and run (<kbd>cmsRun ...</kbd>).</ol></p>  <p>You should get an output file <kbd>DoubleMu.root</kbd>, which contains histograms for 10000 input events (a small subset of the data). These can be looked at using a ROOT Browser. The most interesting histogram is <kbd>GM_mass_log</kbd>. In order to compare it with the invariant-mass spectrum of di-muons in <a href=\"https://inspirehep.net/record/1118729/plots\">MUO-10-004</a>, it should be viewed with the <kbd>logy</kbd> option. With 10000 events the comparison is poor, but the J/&psi; (at log10(mass)=0.5), &Upsilon; (at log10(mass=0.98)) and Z (at log10(mass)=1.95, one event only) peaks should be visible. Furthermore, since the triggers for 2010 and 2011, as well as SingleMu and DoubleMu are different, the general shape of the distributions will differ somewhat depending on input.</p> <p>To run over more or even the full statistics, edit the relevant parts of the Python file <kbd>demoanalyzer_cfg.py</kbd> (see comments therein, change max. number of events and/or insert the proper index and JSON files for 2011/2012 single/dimuon) and rerun. Add up the output histograms from different (nonoverlapping, i.e. use either DoubleMu or SingleMu, not both) input index files using Root tools\n</p><p>In addition to these instructions, which guide you through the example in detail, a github repository (<a href=\"https://github.com/cms-opendata-analyses/DimuonSpectrum2011\">https://github.com/cms-opendata-analyses/DimuonSpectrum2011</a>) based on this original code with minor modifications for direct download is available. </p>  "
```

Comment:
I think you may need to remove one `</li>` right before "Only the first index file is actually needed for initial running", because it does a line break. If that was your intention, then you just need to move the full stop before the `</li>`, because now the new line starts like this: ". Only the first index file is actually..."

Reply:
Also, the second to last paragraph is missing a full stop.

Reply:
OK, right, thanks! Fixed now